android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java

```diff
@@ -2566,7 +2609,8 @@ private static void groupAnnotations(MethodCall call, MethodChannel.Result resul
                 JSONObject annotationJson = new JSONObject(annotation);
                 String primaryAnnotationId = annotationJson.getString(KEY_ANNOTATION_ID);
                 int primaryAnnotPageNum = annotationJson.getInt(KEY_PAGE_NUMBER);
-                Annot primaryAnnotation = ViewerUtils.getAnnotById(pdfViewCtrl, primaryAnnotationId, primaryAnnotPageNum);
+                Annot primaryAnnotation = ViewerUtils.getAnnotById(pdfViewCtrl, primaryAnnotationId,
```

Comment:
there are many strange formatting changes in this file, not sure what's happening

Reply:
maybe it was done by a linter when Kevin was making changes to the file, should i try and revert them all?

Reply:
try auto-reformat see if that fixes it, otherwise yes you will need to revert and do proper formatting using the formatter